packages/resin-event-log/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
+* Mixpanel users will be created if they do not exist on `login`/`signup` in Node (this previously only happened in the browser).
+* Mixpanel is now bundled, and loaded synchronously, so should never fail to load.
```

Comment:
No, you're not wrong - this will make the bundle a little bigger, although I'd expect only very slightly. It should actually decrease the _overall_ download size and the total number of requests we have to make on page load, since we were loading this content anyway, but still, the blocking bundle download itself will be slightly bigger. We should definitely measure the exact increase once this is in the UI, but I'm expecting this to be well worth the small cost.